### PR TITLE
Rename the crates for a bit of reorganisation

### PR DESCRIPTION
**Pull request by BudgieInWA**
What do you think of these renames?

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [workspace]
 members = [
-    "import_streets",
-    "osm2streets",
+    "streets_reader",
```
Comment (BudgieInWA) [inline Cargo.toml:3]: `streetsio`?

Comment (dabreegster) [reply Cargo.toml:3]: Maybe? Or `osm_reader` to emphasize the source, but maybe that just repeats the main crate name? Not opinionated here

**Review by dabreegster — APPROVED** [review]
I'm a fan of the changes! `tests-web` incorrectly emphasized tests. And making the main crate match the project name is great

File: Cargo.toml (hunk 1 above)
Comment (BudgieInWA) [reply Cargo.toml:3]: I like `streetsio` as an analogy to [osmio](https://crates.io/crates/osmio), but I'll leave it as is until its context in the wider ecosystem becomes more obvious.

**Pull request merged**